src/app/settings/stix-objects/sensors/sensor-edit/sensor-edit.component.ts

```diff
@@ -0,0 +1,57 @@
+import { Component, OnInit } from '@angular/core';
+import { Router, ActivatedRoute, Params } from '@angular/router';
+import { Location } from '@angular/common';
+import { MdDialog, MdDialogRef, MdDialogConfig, MdSnackBar } from '@angular/material';
+import { SensorComponent } from '../sensor/sensor.component';
+import { StixService } from '../../../stix.service';
+import { Constance } from '../../../../utils/constance';
+
+@Component({
+  selector: 'sensor-edit',
+  templateUrl: './sensor-edit.component.html'
+})
+
+export class SensorEditComponent extends SensorComponent implements OnInit {
+
+    constructor(
+        public stixService: StixService,
+        public route: ActivatedRoute,
+        public router: Router,
+        public dialog: MdDialog,
+        public location: Location,
+        public snackBar: MdSnackBar) {
+        super(stixService, route, router, dialog, location, snackBar);
+    }
+
+    public ngOnInit() {
+       let sub = super.get().subscribe(
```

Comment:
I think we need to agree on using const, or easing the linter to not complain on non reassigned let.